example/lib/main.dart

```diff
@@ -622,6 +623,78 @@ class _HomePageState extends State<HomePage> {
                 ),
               ],
             ),
+            const SizedBox(height: 5.0),
+            const Text("CheckBoxRow", style: TUITextStyle.heading6),
+            const SizedBox(height: 5.0),
+            TUICheckBoxRow(
+              enableMixedState: true,
+              title: "Title",
+              onChanged: (state) {
+                print("The tapped index is $state");
+              },
+            ),
+            TUICheckBoxRow(
+              enableMixedState: true,
+              title: "Title",
+              backgroundDark: true,
+              onChanged: (state) {
+                print("The tapped index is $state");
+              },
+            ),
+            TUICheckBoxRow(
+              enableMixedState: true,
+              title: "Title",
+              description: "Description",
+              onChanged: (state) {
+                print("The tapped index is $state");
+              },
+            ),
+            TUICheckBoxRow(
+              enableMixedState: true,
+              title: "Title",
+              description: "Description",
+              backgroundDark: true,
+              onChanged: (state) {
+                print("The tapped index is $state");
+              },
+            ),
+            TUICheckBoxRow(
+              enableMixedState: true,
+              title: "Title",
+              state: TUICheckBoxRowState.checked,
+              onChanged: (state) {
+                print("The tapped index is $state");
+              },
+            ),
+            TUICheckBoxRow(
+              enableMixedState: true,
+              title: "Title",
+              backgroundDark: true,
+              state: TUICheckBoxRowState.checked,
+              onChanged: (state) {
+                print("The tapped index is $state");
+              },
+            ),
+            TUICheckBoxRow(
+              enableMixedState: true,
+              title: "Title",
+              description: "Description",
+              state: TUICheckBoxRowState.checked,
+              onChanged: (state) {
+                print("The tapped index is $state");
+              },
+            ),
+            TUICheckBoxRow(
+              enableMixedState: true,
+              title: "Title",
+              description: "Description",
+              backgroundDark: true,
+              state: TUICheckBoxRowState.checked,
+              onChanged: (state) {
+                print("The tapped index is $state");
+              },
+            ),
+            const SizedBox(height: 5.0),
```

Comment:
Please use a consistent vertical spacing of 8.0 pixels as we in the other parts of the example app.